sandcastle/api.py

```diff
@@ -609,8 +608,11 @@ def exec(
                 elif status == "Failure":
                     logger.info("exec command failed")
                     logger.debug(j)
-                    logger.error(f"output = {response!r}")
-                    self._copy_mdir_from_pod(unique_dir)
+                    logger.info(f"output:\n{response}")
```

Comment:
I'm not sure if it's important to consider this also from a non-packit-service POV: a user of the Sandcastle API might want to still get the content of the working directory for inspection, even if the command failed. Wouldn't be more reliable to check the status of the pod in `_copy_mdir_from_pod`? This check would also make the "Success" branch more reliable (where a similar thing could also happen: the pod is shut down after the command was successful but before a copy operation starts).

Reply:
yes, that's a good point, I can change it that way, though to avoid race conditions, I think we should check if the pod is alive after we run `oc rsync`